lighthouse-cli/performance-experiment/report/scripts/perf-x.js

```diff
@@ -31,7 +31,8 @@ class ConfigPanel {
     this._messageField = this._configPanel.querySelector('.js-message');
     this._urlBlockingList = this._configPanel.querySelector('.js-url-blocking-patterns');
     this._urlBlockingStatus = {};
-    this._reportId = new URL(window.location).searchParams.get('id');
+    const match = location.search.match(/[?&]id=([^&]*)/);
```

Comment:
can you add a comment here?  `// Compat: URL.searchParams isn't yet supported by _______ `
so later when that browser updates we'll know when we can remove it